examples/oak.ts

```diff
@@ -72,20 +69,7 @@ const app = new Application<AppState>();
 app.keys = ["super-secret-key"];
 
 // Set up the session middleware
-const sessionStore = new MemoryStore();
-app.use(Session.initMiddleware(sessionStore, {
-  cookieSetOptions: {
-    httpOnly: true,
-    sameSite: "lax",
-    // Enable for when running outside of localhost
-    // secure: true,
-    signed: true,
-  },
-  cookieGetOptions: {
-    signed: true,
-  },
-  expireAfterSeconds: 60 * 10,
-}));
+app.use(Session.initMiddleware() as unknown as Middleware<AppState>);
```

Comment:
This is a quick and dirty solution. Frankly, I haven't even tested if this works. I haven't yet had the time to look at a proper solution.